src/Netscript/RamCostGenerator.ts

```diff
@@ -43,6 +43,7 @@ export const RamCostConstants = {
   BuySellStock: 2.5,
   GetPurchaseServer: 0.25,
   PurchaseServer: 2.25,
+  GetPurchasedServers: 1.05,
```

Comment:
Honestly, these constants are dumb (and used inconsistently). Just go ahead and put the value in directly.

It does explain why it was 2.25 before though, they were all reusing the same value. Which makes me feel less bad about changing it.

Reply:
Updated!